packages/core/src/lib/toast/toast-banner.svelte

```diff
@@ -0,0 +1,119 @@
+<svelte:options immutable />
+
+<script lang="ts">
+import cx from 'classnames';
+import { createEventDispatcher } from 'svelte';
+import { Button, Icon, type IconName } from '$lib';
+import { ToastVariant, type ToastVariantType } from './variants';
+
+/** The type of toast banner to be displayed to the user */
+export let variant: ToastVariantType;
+
+/** Whether there is a close icon displayed */
+export let closeable: boolean | undefined = false;
+
+/** Whether there is an action button displayed */
+export let action = '';
+
+/** The scaling applied on the x axis of the page*/
+export let progress = 1;
```

Comment:
Without a `process` variant, do we need this prop?

Reply:
Removing it